README.markdown

```diff
@@ -114,7 +114,16 @@ services:
 <details>
 <summary>📡 Use <code>network_mode: host</code> (as a hack) to enable IPv6 or read more.</summary>
 
-The setting `network_mode: host` is for IPv6. If you wish to keep the network separated from the host network, check out the [proper way to enable IPv6 support](https://docs.docker.com/config/daemon/ipv6/) and [this GitHub issue about using IPv6 in Docker.](https://github.com/favonia/cloudflare-ddns/issues/119)
+The setting `network_mode: host` is to enable IPv6 by using the host network directly, assuming that the host itself supports IPv6. If you wish to keep the network separated from the host network, check out the [proper way to enable IPv6 support](https://docs.docker.com/config/daemon/ipv6/) and [this GitHub issue about using IPv6 in Docker.](https://github.com/favonia/cloudflare-ddns/issues/119) Please note that the complete support of IPv6 is still under development! Here is a Docker configuration (as `/etc/docker/daemon.json` on Linux) that might work:
```

Comment:
I would say to quickly use the container with IPv6 support is to use the host network instead of the bridge network or read futher how to properly enable IPv6 like IPv4 for all your containers in your bridge network.

Then i would just drop the two links and go straight by explaining the config below.

To enable routeable IPv6 alongside IPv4. Create the following config in: /etc/docker/daemon.json on linux.

This is more clearer to the reader imo. If someone has a mac or windows we can always append that below if there's a request for that (tho i hope one day we can remove the experimental flag).

Reply:
oh, almost forgot! They have to restart the docker daemon with:
sudo systemctl restart docker.service
to apply the changes.

Reply:
Thanks.